!SESSION 2024-01-23 16:37:48.806 -----------------------------------------------
eclipse.buildId=Version 1.14.0
java.version=17.0.8.1
java.vendor=Eclipse Adoptium
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY com.st.stm32cube.ide.mcu.informationcenter 4 4 2024-01-23 16:38:07.791
!MESSAGE CubeMX plugin appears to be active, Log4j initialization might be too late.

!ENTRY com.st.stm32cube.ide.mcu.informationcenter 1 1 2024-01-23 16:38:07.804
!MESSAGE Log4j2 initialized with config file U:\ECE395\exercise3\workspace\.metadata\.log4j2.xml

!ENTRY com.st.stm32cube.ide.mcu.ide 1 1 2024-01-23 16:38:10.185
!MESSAGE Started RMI Server, listening on port 41338

!ENTRY org.eclipse.cdt.core 1 0 2024-01-23 16:39:29.601
!MESSAGE Indexed 'exercise3' (20 sources, 81 headers) in 2.42 sec: 4,279 declarations; 23,871 references; 0 unresolved inclusions; 1 syntax errors; 2 unresolved names (0.0071%)
!SESSION 2024-01-26 12:59:06.280 -----------------------------------------------
eclipse.buildId=Version 1.14.0
java.version=17.0.8.1
java.vendor=Eclipse Adoptium
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY com.st.stm32cube.ide.mcu.informationcenter 4 4 2024-01-26 12:59:14.366
!MESSAGE CubeMX plugin appears to be active, Log4j initialization might be too late.

!ENTRY com.st.stm32cube.ide.mcu.informationcenter 1 1 2024-01-26 12:59:14.377
!MESSAGE Log4j2 initialized with config file U:\ECE395\exercise3\workspace\.metadata\.log4j2.xml

!ENTRY com.st.stm32cube.ide.mcu.ide 1 1 2024-01-26 12:59:18.924
!MESSAGE Started RMI Server, listening on port 41337

!ENTRY org.eclipse.cdt.dsf.gdb 4 5012 2024-01-26 13:23:05.692
!MESSAGE Error in final launch sequence:

Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
!STACK 1
org.eclipse.core.runtime.CoreException: Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.concurrent.Query.get(Query.java:112)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launchDebugSession(GdbLaunchDelegate.java:252)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launchDebugger(GdbLaunchDelegate.java:109)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launch(GdbLaunchDelegate.java:97)
	at com.st.stm32cube.ide.mcu.debug.launch.DSFDelegate.launch(DSFDelegate.java:334)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:805)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:716)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1038)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$1.run(DebugUIPlugin.java:1241)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.lang.Exception: localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.processMIOutput(AbstractMIControl.java:1024)
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.run(AbstractMIControl.java:853)
!SUBENTRY 1 org.eclipse.cdt.dsf.gdb 4 10004 2024-01-26 13:23:05.693
!MESSAGE Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
!STACK 0
java.lang.Exception: localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.processMIOutput(AbstractMIControl.java:1024)
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.run(AbstractMIControl.java:853)

!ENTRY org.eclipse.cdt.dsf.gdb 4 5012 2024-01-26 13:23:39.279
!MESSAGE Error in final launch sequence:

Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
!STACK 1
org.eclipse.core.runtime.CoreException: Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.concurrent.Query.get(Query.java:112)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launchDebugSession(GdbLaunchDelegate.java:252)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launchDebugger(GdbLaunchDelegate.java:109)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launch(GdbLaunchDelegate.java:97)
	at com.st.stm32cube.ide.mcu.debug.launch.DSFDelegate.launch(DSFDelegate.java:334)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:805)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:716)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1038)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$1.run(DebugUIPlugin.java:1241)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.lang.Exception: localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.processMIOutput(AbstractMIControl.java:1024)
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.run(AbstractMIControl.java:853)
!SUBENTRY 1 org.eclipse.cdt.dsf.gdb 4 10004 2024-01-26 13:23:39.279
!MESSAGE Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
!STACK 0
java.lang.Exception: localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.processMIOutput(AbstractMIControl.java:1024)
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.run(AbstractMIControl.java:853)

!ENTRY org.eclipse.cdt.dsf.gdb 4 5012 2024-01-26 13:24:47.613
!MESSAGE Error in final launch sequence:

Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
!STACK 1
org.eclipse.core.runtime.CoreException: Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.concurrent.Query.get(Query.java:112)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launchDebugSession(GdbLaunchDelegate.java:252)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launchDebugger(GdbLaunchDelegate.java:109)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launch(GdbLaunchDelegate.java:97)
	at com.st.stm32cube.ide.mcu.debug.launch.DSFDelegate.launch(DSFDelegate.java:334)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:805)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:716)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1038)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$1.run(DebugUIPlugin.java:1241)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.lang.Exception: localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.processMIOutput(AbstractMIControl.java:1024)
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.run(AbstractMIControl.java:853)
!SUBENTRY 1 org.eclipse.cdt.dsf.gdb 4 10004 2024-01-26 13:24:47.613
!MESSAGE Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
!STACK 0
java.lang.Exception: localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.processMIOutput(AbstractMIControl.java:1024)
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.run(AbstractMIControl.java:853)

!ENTRY org.eclipse.cdt.dsf.gdb 4 5012 2024-01-26 13:33:46.223
!MESSAGE Error in final launch sequence:

Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
!STACK 1
org.eclipse.core.runtime.CoreException: Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.concurrent.Query.get(Query.java:112)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launchDebugSession(GdbLaunchDelegate.java:252)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launchDebugger(GdbLaunchDelegate.java:109)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launch(GdbLaunchDelegate.java:97)
	at com.st.stm32cube.ide.mcu.debug.launch.DSFDelegate.launch(DSFDelegate.java:334)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:805)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:716)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1038)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$1.run(DebugUIPlugin.java:1241)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.lang.Exception: localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.processMIOutput(AbstractMIControl.java:1024)
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.run(AbstractMIControl.java:853)
!SUBENTRY 1 org.eclipse.cdt.dsf.gdb 4 10004 2024-01-26 13:33:46.223
!MESSAGE Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
!STACK 0
java.lang.Exception: localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.processMIOutput(AbstractMIControl.java:1024)
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.run(AbstractMIControl.java:853)

!ENTRY org.eclipse.cdt.dsf.gdb 4 5012 2024-01-26 13:34:08.508
!MESSAGE Error in final launch sequence:

Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
!STACK 1
org.eclipse.core.runtime.CoreException: Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.concurrent.Query.get(Query.java:112)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launchDebugSession(GdbLaunchDelegate.java:252)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launchDebugger(GdbLaunchDelegate.java:109)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launch(GdbLaunchDelegate.java:97)
	at com.st.stm32cube.ide.mcu.debug.launch.DSFDelegate.launch(DSFDelegate.java:334)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:805)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:716)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1038)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$1.run(DebugUIPlugin.java:1241)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.lang.Exception: localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.processMIOutput(AbstractMIControl.java:1024)
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.run(AbstractMIControl.java:853)
!SUBENTRY 1 org.eclipse.cdt.dsf.gdb 4 10004 2024-01-26 13:34:08.508
!MESSAGE Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
!STACK 0
java.lang.Exception: localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.processMIOutput(AbstractMIControl.java:1024)
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.run(AbstractMIControl.java:853)

!ENTRY org.eclipse.cdt.dsf.gdb 4 5012 2024-01-26 14:15:23.334
!MESSAGE Error in final launch sequence:

Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
!STACK 1
org.eclipse.core.runtime.CoreException: Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.concurrent.Query.get(Query.java:112)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launchDebugSession(GdbLaunchDelegate.java:252)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launchDebugger(GdbLaunchDelegate.java:109)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launch(GdbLaunchDelegate.java:97)
	at com.st.stm32cube.ide.mcu.debug.launch.DSFDelegate.launch(DSFDelegate.java:334)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:805)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:716)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1038)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$1.run(DebugUIPlugin.java:1241)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.lang.Exception: localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.processMIOutput(AbstractMIControl.java:1024)
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.run(AbstractMIControl.java:853)
!SUBENTRY 1 org.eclipse.cdt.dsf.gdb 4 10004 2024-01-26 14:15:23.334
!MESSAGE Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
!STACK 0
java.lang.Exception: localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.processMIOutput(AbstractMIControl.java:1024)
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.run(AbstractMIControl.java:853)

!ENTRY org.eclipse.cdt.dsf.gdb 4 5012 2024-01-26 14:16:05.045
!MESSAGE Error in final launch sequence:

Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
!STACK 1
org.eclipse.core.runtime.CoreException: Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.concurrent.Query.get(Query.java:112)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launchDebugSession(GdbLaunchDelegate.java:252)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launchDebugger(GdbLaunchDelegate.java:109)
	at org.eclipse.cdt.dsf.gdb.launching.GdbLaunchDelegate.launch(GdbLaunchDelegate.java:97)
	at com.st.stm32cube.ide.mcu.debug.launch.DSFDelegate.launch(DSFDelegate.java:334)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:805)
	at org.eclipse.debug.internal.core.LaunchConfiguration.launch(LaunchConfiguration.java:716)
	at org.eclipse.debug.internal.ui.DebugUIPlugin.buildAndLaunch(DebugUIPlugin.java:1038)
	at org.eclipse.debug.internal.ui.DebugUIPlugin$1.run(DebugUIPlugin.java:1241)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.lang.Exception: localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.processMIOutput(AbstractMIControl.java:1024)
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.run(AbstractMIControl.java:853)
!SUBENTRY 1 org.eclipse.cdt.dsf.gdb 4 10004 2024-01-26 14:16:05.045
!MESSAGE Failed to execute MI command:
target remote localhost:61234

Error message from debugger back end:
localhost:61234: Connection timed out.
!STACK 0
java.lang.Exception: localhost:61234: Connection timed out.
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.processMIOutput(AbstractMIControl.java:1024)
	at org.eclipse.cdt.dsf.mi.service.command.AbstractMIControl$RxThread.run(AbstractMIControl.java:853)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:12:31.021
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.markFileIncomplete(Database.java:836)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:801)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:560)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:12:31.055
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:977)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:572)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:12:31.055
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:621)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:12:31.055
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:784)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1549)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:195)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:169)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:623)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:12:37.134
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:560)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:12:37.140
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.removeFilesInIndex(AbstractIndexerTask.java:863)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:563)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:12:37.187
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:977)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:572)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:12:37.193
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:621)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:12:37.199
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:784)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1549)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:195)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:169)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:623)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:06.092
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:560)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:06.114
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:977)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:572)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:06.121
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:621)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:06.126
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:784)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1549)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:195)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:169)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:623)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:10.902
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:560)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:10.908
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.removeFilesInIndex(AbstractIndexerTask.java:863)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:563)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:10.918
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:621)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:10.923
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:784)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1549)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:195)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:169)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:623)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:13.398
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:560)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:13.421
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:977)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:572)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:13.427
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:621)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:13.433
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:784)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1549)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:195)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:169)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:623)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:59.874
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:560)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:59.900
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:977)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:572)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:59.907
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:621)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:13:59.912
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:784)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1549)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:195)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:169)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:623)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:04.852
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:560)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:04.887
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:977)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:572)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:04.893
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:621)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:04.900
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:784)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1549)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:195)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:169)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:623)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:06.456
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:560)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:06.462
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.removeFilesInIndex(AbstractIndexerTask.java:863)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:563)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:06.467
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:621)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:06.473
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:784)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1549)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:195)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:169)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:623)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:08.369
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:560)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:08.394
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:977)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:572)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:08.402
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:621)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:08.408
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:784)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1549)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:195)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:169)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:623)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:19.439
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:560)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:19.472
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:977)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:572)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:19.478
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:754)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1127)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:145)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:130)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:642)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:621)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.cdt.core 4 0 2024-01-30 14:20:19.484
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at java.base/sun.nio.ch.FileDispatcherImpl.pwrite(FileDispatcherImpl.java:74)
	at java.base/sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:130)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:97)
	at java.base/sun.nio.ch.IOUtil.write(IOUtil.java:67)
	at java.base/sun.nio.ch.FileChannelImpl.writeInternal(FileChannelImpl.java:878)
	at java.base/sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:861)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:193)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:55)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:806)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:784)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1549)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:195)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:169)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:623)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:164)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:160)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.ui 4 4 2024-01-30 14:53:52.992
!MESSAGE Unable to create part
!STACK 1
org.eclipse.ui.PartInitException: Invalid Input: Must be IFileEditorInput
	at com.st.stm32cube.common.mx.editor.CubeMxEditor.init(CubeMxEditor.java:834)
	at org.eclipse.ui.internal.EditorReference.initialize(EditorReference.java:353)
	at org.eclipse.ui.internal.e4.compatibility.CompatibilityPart.create(CompatibilityPart.java:344)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.e4.core.internal.di.MethodRequestor.execute(MethodRequestor.java:58)
	at org.eclipse.e4.core.internal.di.InjectorImpl.processAnnotated(InjectorImpl.java:971)
	at org.eclipse.e4.core.internal.di.InjectorImpl.processAnnotated(InjectorImpl.java:936)
	at org.eclipse.e4.core.internal.di.InjectorImpl.internalInject(InjectorImpl.java:142)
	at org.eclipse.e4.core.internal.di.InjectorImpl.internalMake(InjectorImpl.java:386)
	at org.eclipse.e4.core.internal.di.InjectorImpl.make(InjectorImpl.java:313)
	at org.eclipse.e4.core.contexts.ContextInjectionFactory.make(ContextInjectionFactory.java:202)
	at org.eclipse.e4.ui.internal.workbench.ReflectionContributionFactory.createFromBundle(ReflectionContributionFactory.java:91)
	at org.eclipse.e4.ui.internal.workbench.ReflectionContributionFactory.doCreate(ReflectionContributionFactory.java:60)
	at org.eclipse.e4.ui.internal.workbench.ReflectionContributionFactory.create(ReflectionContributionFactory.java:42)
	at org.eclipse.e4.ui.workbench.renderers.swt.ContributedPartRenderer.createWidget(ContributedPartRenderer.java:132)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.createWidget(PartRenderingEngine.java:995)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.safeCreateGui(PartRenderingEngine.java:659)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.safeCreateGui(PartRenderingEngine.java:763)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$2.run(PartRenderingEngine.java:728)
	at org.eclipse.core.runtime.SafeRunner.run(SafeRunner.java:45)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.createGui(PartRenderingEngine.java:712)
	at org.eclipse.e4.ui.internal.workbench.PartServiceImpl.lambda$0(PartServiceImpl.java:105)
	at org.eclipse.e4.ui.services.internal.events.UIEventHandler.lambda$0(UIEventHandler.java:38)
	at org.eclipse.swt.widgets.Synchronizer.syncExec(Synchronizer.java:183)
	at org.eclipse.ui.internal.UISynchronizer.syncExec(UISynchronizer.java:133)
	at org.eclipse.swt.widgets.Display.syncExec(Display.java:4803)
	at org.eclipse.e4.ui.workbench.swt.DisplayUISynchronize.syncExec(DisplayUISynchronize.java:34)
	at org.eclipse.e4.ui.services.internal.events.UIEventHandler.handleEvent(UIEventHandler.java:38)
	at org.eclipse.equinox.internal.event.EventHandlerWrapper.handleEvent(EventHandlerWrapper.java:205)
	at org.eclipse.equinox.internal.event.EventHandlerTracker.dispatchEvent(EventHandlerTracker.java:203)
	at org.eclipse.equinox.internal.event.EventHandlerTracker.dispatchEvent(EventHandlerTracker.java:1)
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:234)
	at org.eclipse.osgi.framework.eventmgr.ListenerQueue.dispatchEventSynchronous(ListenerQueue.java:151)
	at org.eclipse.equinox.internal.event.EventAdminImpl.dispatchEvent(EventAdminImpl.java:133)
	at org.eclipse.equinox.internal.event.EventAdminImpl.sendEvent(EventAdminImpl.java:75)
	at org.eclipse.equinox.internal.event.EventComponent.sendEvent(EventComponent.java:44)
	at org.eclipse.e4.ui.services.internal.events.EventBroker.send(EventBroker.java:55)
	at org.eclipse.e4.ui.internal.workbench.UIEventPublisher.notifyChanged(UIEventPublisher.java:63)
	at org.eclipse.emf.common.notify.impl.BasicNotifierImpl.eNotify(BasicNotifierImpl.java:424)
	at org.eclipse.e4.ui.model.application.ui.impl.ElementContainerImpl.setSelectedElementGen(ElementContainerImpl.java:170)
	at org.eclipse.e4.ui.model.application.ui.impl.ElementContainerImpl.setSelectedElement(ElementContainerImpl.java:188)
	at org.eclipse.e4.ui.internal.workbench.ModelServiceImpl.showElementInWindow(ModelServiceImpl.java:661)
	at org.eclipse.e4.ui.internal.workbench.ModelServiceImpl.bringToTop(ModelServiceImpl.java:625)
	at org.eclipse.e4.ui.internal.workbench.PartServiceImpl.delegateBringToTop(PartServiceImpl.java:796)
	at org.eclipse.e4.ui.internal.workbench.PartServiceImpl.showPart(PartServiceImpl.java:1267)
	at org.eclipse.ui.internal.WorkbenchPage.busyOpenEditor(WorkbenchPage.java:3233)
	at org.eclipse.ui.internal.WorkbenchPage.lambda$11(WorkbenchPage.java:3123)
	at org.eclipse.swt.custom.BusyIndicator.showWhile(BusyIndicator.java:67)
	at org.eclipse.ui.internal.WorkbenchPage.openEditor(WorkbenchPage.java:3121)
	at org.eclipse.ui.internal.WorkbenchPage.openEditor(WorkbenchPage.java:3091)
	at org.eclipse.ui.internal.WorkbenchPage.openEditor(WorkbenchPage.java:3074)
	at org.eclipse.ui.ide.IDE.openEditorOnFileStore(IDE.java:1340)
	at org.eclipse.ui.internal.ide.actions.OpenLocalFileAction.run(OpenLocalFileAction.java:95)
	at org.eclipse.ui.internal.ide.actions.OpenLocalFileAction.run(OpenLocalFileAction.java:68)
	at org.eclipse.ui.internal.PluginAction.runWithEvent(PluginAction.java:239)
	at org.eclipse.ui.internal.WWinPluginAction.runWithEvent(WWinPluginAction.java:218)
	at org.eclipse.jface.action.ActionContributionItem.handleWidgetSelection(ActionContributionItem.java:580)
	at org.eclipse.jface.action.ActionContributionItem.lambda$4(ActionContributionItem.java:414)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4274)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1066)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4072)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3660)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:645)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:342)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:552)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:171)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:152)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:651)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:588)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1459)
!SUBENTRY 1 org.eclipse.ui 4 0 2024-01-30 14:53:52.992
!MESSAGE Invalid Input: Must be IFileEditorInput
